049-SQLDedicatedPoolPerf/Coach/README.md

```diff
@@ -0,0 +1,51 @@
+# What The Hack - Dedicated SQL Pool performance best practices - Coach Guide
```

Comment:
Please update the title to match the title on the home page as below:

"Synapse Dedicated SQL Pool - Performance Best Practices"